bubble_sort/Python/bubbleSort.py

```diff
@@ -0,0 +1,24 @@
+#Bubble Sort in Python
+
+array = [] #declaration of array
+
+
+#getting f user inputs and append that values to the array
+for x in range(5):
+    array.append(eval(input("number " + "")))
+
+n = len(array) #array length
```

Comment:
E262 inline comment should start with '# '

*Origin: PycodestyleBear (E262), Section: `all.autopep8`.*